core/src/allocation_init.jl

```diff
@@ -182,7 +182,10 @@ function add_flow!(
 end
 
 """
-Equate the inflow and outflow of conservative nodes
+Add flow conservation constraints for conservative nodes.
+Ensures that inflow equals outflow for nodes that conserve mass (pumps, outlets, 
```

Comment:
```suggestion
Ensures that inflow equals outflow for nodes that conserve mass (pumps, outlets,
```

The trailing whitespace here fails our tests. We use pre-commit to automatically strip these, which can be configured once with `pixi run install`.

Reply:
Thanks, @visr